db/migrate/20200817030307_make_contact_nullable_on_tickets.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class MakeContactNullableOnTickets < ActiveRecord::Migration[6.0]
+  def change
+    change_column :tickets, :contact_id, :bigint, null: true
```

Comment:
oops

Reply:
Ahaha yeah this was a mistake on our part, but making it better!